src/sass/base/_fixes.scss

```diff
@@ -0,0 +1,6 @@
+// Do not display an error spacer underneath input/textarea fields if it's empty
+.md-errors-spacer {
+    &:empty {
+        display: none;
+    }
+}
```

Comment:
Since you're applying this to all md elements no matter where on the page, couldn't that in some cases modify the layout and/or cause layout jumps in case the error spacer is actually being used?

In doubt, just apply it to the dialog in order not to cause side effects. Otherwise, make sure that we're not relying on this anywhere. (It's well possible that it has no unintended side effects whatsoever, I'm just saying that we should double-check :slightly_smiling_face:)

Reply:
> Since you're applying this to all md elements no matter where on the page, couldn't that in some cases modify the layout and/or cause layout jumps in case the error spacer is actually being used?

Correct. I can move it into a separate class if you want me to.

Reply:
Yeah, maybe `md-skip-error-spacer` or something like that.